docs/Operators.md

```diff
@@ -885,47 +887,72 @@ Equation definitions:
 `h` - hidden gate
 `t` - time step (t-1 means previous time step)
 `Xi` - input tensor
-`W[izrfcoh]` - W parameter weight matrices for the corresponding gates
-`R[izrfcoh]` - R parameter weight matrices for the corresponding gates
-`Wb[izrfcoh]` - W parameter bias vectors for the corresponding gates
-`Rb[izrfcoh]` - R parameter bias vectors for the corresponding gates
+`W[izrfcohp]` - W parameter weight matrices for the corresponding gates
+`R[izrfcohp]` - R parameter weight matrices for the corresponding gates
+`Wb[izrfcohp]` - W parameter bias vectors for the corresponding gates
+`Rb[izrfcohp]` - R parameter bias vectors for the corresponding gates
 `ReLU(X)` - max(X, 0)
 `tanh` - hyperbolic tangent of X
 `sigmoid(X)` - 1 / (1 + e^-X)
 `[C|H]` - Cell/Hidden state
 
 - Equations:
-  `relu`
+  `simple(relu)`
   - Ht = ReLU(Wi*Xt + Ri*Ht-1 + Wbi + Rbi)
-  `tanh`
+
+  `simple(tanh)`
   - Ht = tanh(Wi*Xt + Ri*Ht-1 + Wbi + Rbi)
-  `lstm`
+
+  `lstm with default activations`
   - it = sigmoid(Wi*Xt + Ri*Ht-1 + Wbi + Rbi)
   - ft = sigmoid(Wf*Xt + Rf*Ht-1 + Wbf + Rbf)
   - ot = sigmoid(Wo*Xt + Ro*Ht-1 + Wbo + Rbo)
   - ct = tanh(Wc*Xt + Rc*Ht-1 + Wbc + Rbc)
-  - C = ft * Ct-1 + it * ct
-  - H = ot * tanh(C)
-  `gru`
+  - Ct = ft (.) Ct-1 + it (.) ct
+  - H = ot (.) tanh(C)
+
+  `gru with default activations`
   - zt = sigmoid(Wz*Xt + Rz*Ht-1 + Wbz + Rbz)
   - rt = sigmoid(Wr*Xt + Rr*Ht-1 + Wbr + Rbr)
   - ht = tanh(Wh*Xt + rt *(Rh*Ht-1 + Rbh) + Wbh)
-  - H = (1 - zt) * ht + it * Ht-1
+  - H = (1 - zt) (.) ht + it (.) Ht-1
+
+  `lstm_peephole with default activations`
+  - it = sigmoid(Wi*Xt + Ri*Ht-1 + Pi(.)Ct-1 + Wbi + Rbi)
+  - ft = sigmoid(Wf*Xt + Rf*Ht-1 + Pf(.)Ct-1 + Wbf + Rbf)
+  - ot = sigmoid(Wo*Xt + Ro*Ht-1 + Po(.)Ct-1 + Wbo + Rbo)
+  - ct = tanh(Wc*Xt + Rc*Ht-1 + Wbc + Rbc)
+  - Ct = ft (.) Ct-1 + it (.) ct
+  - H = ot (.) tanh(C)
+
+  `lstm_input_forget with default activations`
+  - it = sigmoid(Wi*Xt + Ri*Ht-1 + Wbi + Rbi)
+  - ft = sigmoid(Wf*Xt + Rf*Ht-1 + Wbf + Rbf)
```

Comment:
This is not needed anymore. We may also comment that forget weights are ignored for this type of cell

Reply:
Good catch. Removed.